go/vt/vttablet/tabletmanager/vreplication/vplayer.go

```diff
@@ -171,6 +171,19 @@ func (vp *vplayer) fetchAndApply(ctx context.Context) error {
 	}
 }
 
+func (vp *vplayer) applyStmtEvent(ctx context.Context, event *binlogdatapb.VEvent) error {
+	for _, rule := range vp.vr.source.Filter.Rules {
```

Comment:
This could be precomputed as a member of vplayer.

Reply:
Could you elaborate here? I did not follow this comment.

Reply:
I meant that you can run this loop once in the beginning and save the result as a member of vplayer, like `canAcceptStmtEvents`. Otherwise, you'll be looping for every statement.

Reply:
Ah I see. Fixed.